charter/index.md

```diff
@@ -0,0 +1,219 @@
+
+This group maintains an [index](https://font-text.github.io/work-items/)
+of the documents it is working on.
+
+## Communication
+
+This group operates under the
+[W3C Code of Ethics and Professional Conduct](https://www.w3.org/Consortium/cepc/).
+
+This group will conduct its work publicly,
+primarily using the public-font-text@w3.org mailing list
+([archives](https://lists.w3.org/Archives/Public/public-font-text/))
+for minutes, announcements, and high level discussions,
+and [GitHub](https://github.com/font-text)
+for detailed discussions
+(See [Contribution Mechanics](#contribution-mechanics)).)
+
+Teleconferences and face-to-face meetings
+may be held from time to time,
+provided sufficient advance notice is given.
+Minutes of any such meeting will be
+posted to the group’s public mailing list
+in a timely manner,
+and will include
+a transcript of the meeting
+to the best of the minute-taker's ability,
+together with a list of any
+decisions and action items resolved within the meeting
+(See [Decision Policy](#decision-policy)).
+
+Deliverables and supporting documents
+produced by this Community Group are
+hosted on the group’s [GitHub](https://github.com/font-text).
+
+The group uses the private internal-font-text@w3.org mailing list ([archives](https://lists.w3.org/Archives/Member/internal-font-text/))
+for administrative purposes,
+such as organizing meetings,
+requesting commit access to GitHub,
+etc.
+
+Additionally, the group will use
+the #font-text channel on the [W3C IRC server](https://www.w3.org/Project/IRC/)
+during meetings
+and for day-to-day casual chatting.
+
+## Decision Policy
+
+This group will seek to make decisions when there is consensus,
+and favors proposals that create the weakest objections.
+This is preferred over proposals that are supported by a large majority
+but that cause strong objections from a few people.
+
+In some cases,
+even after careful consideration of all points of view,
+the group might find itself unable to reach consensus.
+The Chair(s) may record a decision where there is dissent,
+while minuting objections,
+so that the group can make progress.
+Dissenters cannot stop a group’s work
+simply by saying that they cannot live with a decision.
+When the Chair(s) believe that the Group has
+duly considered the legitimate concerns of dissenters
+as far as is possible and reasonable,
+the group should move on.
+
+In addition to decisions made
+on teleconferences or face to face meetings,
+decisions may also be made by a call for consensus
+on the public mailing list;
+consensus to be determined by the Chair(s)
+after some reasonable interval for objections.
+
+All decisions made by the group should be considered resolved
+unless and until new information becomes available
+or unless reopened at the discretion of the Chair(s).
+
+The Chair(s) may appoint individuals to be the
+Editors of the Community Group's Deliverables.
+Such Editors are bound by the consensus of the Working Group
```

Comment:
```suggestion
Such Editors are bound by the consensus of the Community Group
```

(I scrapped this catch from the [meeting minutes Google Doc](https://docs.google.com/document/d/1KoknOb0IMAPeiLhifvc_AqB7s3rs6VKOsTQg_oCDypQ/edit?usp=sharing), credit appears to be due @svgeesus.)

Reply:
Resolved manually in c802e4e.